### PR TITLE
[LOOP-1838] Display correct active carbs widget is updated (even if added via the watch)

### PR DESCRIPTION
**Pull request by nhamming**
https://tidepool.atlassian.net/browse/LOOP-1838

Instead of pulling cob from the store, use the value provided by the loop state (similar to how other parameters of the status extension context are set).

Note the number of digits used to display the IOB was changed to match how this value is displayed in the app.

### CODE REVIEW

File: Common/Models/StatusExtensionContext.swift [hunk 1]
```diff
@@ -293,6 +293,7 @@ struct StatusExtensionContext: RawRepresentable {
     var pumpLifecycleProgressContext: DeviceLifecycleProgressContext?
     var cgmStatusHighlightContext: DeviceStatusHighlightContext?
     var cgmLifecycleProgressContext: DeviceLifecycleProgressContext?
+    var carbsOnBoardValue: Double?
```
Comment (darinkrauss) [inline Common/Models/StatusExtensionContext.swift]: [nit] Any reason for `Value` suffix?

File: Loop Status Extension/StatusViewController.swift [hunk 2]
```diff
@@ -92,15 +92,6 @@ class StatusViewController: UIViewController, NCWidgetProviding {
         insulinSensitivitySchedule: defaults?.insulinSensitivitySchedule
     )
     
-    let absorptionTimes = LoopSettings.defaultCarbAbsorptionTimes
-    lazy var carbStore = CarbStore(healthStore: healthStore,
```
Comment (darinkrauss) [inline Loop Status Extension/StatusViewController.swift:96]: 👍

File: Loop Status Extension/StatusViewController.swift [hunk 3]
```diff
@@ -263,8 +242,8 @@ class StatusViewController: UIViewController, NCWidgetProviding {
                 let numberFormatter = NumberFormatter()
 
                 numberFormatter.numberStyle = .decimal
-                numberFormatter.minimumFractionDigits = 1
-                numberFormatter.maximumFractionDigits = 1
+                numberFormatter.minimumFractionDigits = 2
+                numberFormatter.maximumFractionDigits = 2
```
Comment (darinkrauss) [inline Loop Status Extension/StatusViewController.swift:246]: This looks unrelated to this change. Intentional?

**Review by darinkrauss — APPROVED** [review]
A couple of comments for your review, but otherwise LGTM! 🚀

File: Common/Models/StatusExtensionContext.swift (hunk 1 above)
Comment (nhamming) [reply Common/Models/StatusExtensionContext.swift]: my intention here is so that it isn't confused with a value & unit pairing. But I see that didn't help. :) I'll remove.

File: Loop Status Extension/StatusViewController.swift (hunk 3 above)
Comment (nhamming) [reply Loop Status Extension/StatusViewController.swift:246]: Yes, I added a note in the PR description that this change is such that the IOB displayed in the app matches that displayed in the widget.

Comment (darinkrauss) [reply Loop Status Extension/StatusViewController.swift:246]: Ah, I missed that. :+1:
